docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch a subset of nodes with a user-specified label.
+* Creation of BIG-IP SSL Profiles from Kubernetes Secrets via Ingress TLS.
+* Creation of BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, Edge, Passthrough, and Re-encrypt Routes.
+
+* This is a feature complete upgrade from the OpenShift F5Router. See the documentation for details
```

Comment:
`See [Replace the OpenShift F5 Router with the BIG-IP Controller](http://clouddocs.f5.com/containers/elbert/openshift/replace-f5-router.html) for more information. `